src/components/Inventory/view/InventorySeedlingsTable.tsx

```diff
@@ -129,7 +128,7 @@ export default function InventorySeedslingsTable(props: InventorySeedslingsTable
 
       if (activeRequests) {
         const batchesResults = searchResponse?.map((sr) => {
-          return { ...sr, facilityId: sr.facility_id } as Batch;
+          return { ...sr, facilityId: sr.facility_id };
```

Comment:
search results are not `Batch` types from batch specific api. For example, totalQuantity does not exist in Batch but exists in search results